src/components/atoms/ChatOptionButton.vue

```diff
@@ -12,7 +12,7 @@
       bg-white
       hover:bg-gray-infolt
     "
-    aria-label="Quick reply"
+    :aria-label="$t(text)"
```

Comment:
Adding text in here without translation should be fine as the text will be ideally passed in translated as a prop. The yes/no options can be translated in ConversationWindow.vue line 44 ":buttonOptions="['Yes', 'No']""

Reply:
this is so the voice recorder reads "yes' and 'no" otherwise yes and no are both read as 'quick reply" and that makes no sense.

Reply:
The text itself doesn't need to be translated though

Reply:
Right, of course.